rootfs/opt/gcp/create-gke-cluster.sh

```diff
@@ -5,15 +5,14 @@ set -e
 . "$QSERV_CFG_DIR/env-infra.sh"
 
 # Creates a GKE cluster
-#gcloud auth login
 gcloud config set project "$PROJECT"
-gcloud beta container --project "$PROJECT" clusters create "$CLUSTER" \
-    --zone "$ZONE" --username "admin" --cluster-version "$CLUSTER_VERSION" \
-    --machine-type "$DEFAULT_MTYPE" --image-type "COS" \
-    --disk-type "pd-standard" --disk-size "100" \
-    --scopes $SCOPE \
-    --num-nodes "$SIZE_DEFAULT" --enable-cloud-logging --enable-cloud-monitoring \
+gcloud beta container --project "$PROJECT" clusters create "$CLUSTER" --zone "$ZONE" \
+    --no-enable-basic-auth --cluster-version "1.12.5-gke.5" --machine-type "$MTYPE_WORKER" \
```

Comment:
Yes because the command "gcloud beta container" arguments and options are dependant on cluster version. So I tried to parametrize only stable arguments and options. We need to wait for the "beta" keyword to be removed to have something more stable.